### PR TITLE
added security warning for graphml files

### PR DESCRIPTION
**Pull request by bt-nia**

### CODE REVIEW

File: networkx/readwrite/graphml.py [hunk 1]
```diff
@@ -4,6 +4,10 @@
 *******
 Read and write graphs in GraphML format.
 
+Security Warning: This parser uses the standard xml library present in Python. 
+It is vulnerable to several XML related vulnerabilites, such as XXE and denial 
+of service attacks. Only parse GraphML files you trust.
+
```
Comment (rossbar) [inline networkx/readwrite/graphml.py:10]:
```suggestion
.. warning::

    This parser uses the standard xml library present in Python, which is
    insecure - see :doc:`library/xml` for additional information.
    Only parse GraphML files you trust.

```

**Review by rossbar — COMMENTED** [review]
+1 for adding a security warning, though rather than expound on it ourselves I'd point to the related Python docs instead. I've also suggested using the `.. warning` directive just to match what the Python docs do.

**Comment by bt-nia** [comment]
@rossbar Thanks for looking into it! I agree :)

**Review by rossbar — APPROVED** [review]
Thanks @bt-nia !

**Comment by MridulS** [comment]
Thanks @bt-nia!

**Pull request merged**